charts/keycloak/templates/service-http.yaml

```
@@ -23,6 +23,9 @@ spec:
   loadBalancerSourceRanges:
     {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
+  {{- if and (eq "LoadBalancer" .Values.service.type) .Values.service.externalTrafficPolicy }}
+  externalTrafficPolicy: {{ .Values.service.loadBalancerSourceRanges }}
+  {{- end }}
```

Move default value to values.yaml
```suggestion
  {{- if eq "LoadBalancer" .Values.service.type }}
  externalTrafficPolicy: {{ .Values.service.loadBalancerSourceRanges }}
  {{- end }}
```

---

Updated as requested